test/unit/sequence_base_test.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+require_relative '../test_helper'
+
+class DelayedResourceTestSequence < MiniTest::Test
```

Comment:
Why is this named Delayed Resource Test Sequence if it is in `test/unit/sequence_base_test.rb`?

Reply:
I started writing it as a unit test for just the delayed stuff, but forgot to change the name when I renamed the file. Should be fixed now.